src/GroupAction/GroupActionCollection.php

```diff
@@ -46,17 +46,46 @@ public function addToFormContainer(Container $container): void
 		}
 
 		/**
-		 * First foreach for filling "main" select
+		 * First foreach for adding button actions
 		 */
 		foreach ($this->groupActions as $id => $action) {
-			$main_options[$id] = $action->getTitle();
+			$control = null;
```

Comment:
Is this line necessary?

Reply:
I don't know, it's maybe backdoors for FBI.